src/revivalpmmp/pureentities/entity/monster/flying/Blaze.php

```diff
@@ -165,12 +164,12 @@ public function updateMove($tickDiff) {
                     } else {
                         $this->motionY = 0;
                     }
-                } else {
+                } elseif ($diff > 0) {
                     $this->motionX = $this->getSpeed() * 0.15 * ($x / $diff);
                     $this->motionZ = $this->getSpeed() * 0.15 * ($z / $diff);
                 }
             }
-            $this->yaw = rad2deg(-atan2($x / $diff, $z / $diff));
+            if ($diff > 0) $this->yaw = rad2deg(-atan2($x / $diff, $z / $diff));
```

Comment:
This could be moved to be within the elseif statement

Reply:
Are you sure that wouldn't change the behaviour? If you do that, the yaw code won't be run as it is now if ($x ** 2 + $z ** 2 < 0.5)

Reply:
🤦‍♂️ oops I mixed up which bracket goes to which statement